app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
Yeah, I used a generator to create the Feedback model. I don't know what this does too.